luda-editor/resources/sequence/0일차 아침.json

```diff
@@ -21,16 +21,18 @@
                 ],
                 "source_01_circumscribed": {
                   "center": {
-                    "x": 0.513378,
-                    "y": 0.8983649
+                    "x": 0.5628404,
+                    "y": 0.99394435
                   },
-                  "radius": 0.5
+                  "radius": 0.6046556
```

Comment:
camera_angle has been modified. Is it alright?

Reply:
Yes it's ok. i don't care about this sequence because we will replace this with takiki's animation.